### PR TITLE
Update 0212 add ability to reuse a syncfilename for a putfile with accepted revisions

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Updating with revisions specified here: [#642](https://github.com/smartdevicelink/sdl_evolution/issues/642#issuecomment-447108125)

### CODE REVIEW

**Comment by NicoleYarroch** [comment]
@jordynmackool please review these updates based on the accepted revisions.

File: proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md [hunk 1]
```diff
@@ -37,6 +35,54 @@ When Core receives a `PutFile` it should send a notification to the HMI that a n
...
+    
+    <param name="length" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>
+        Optional length in bytes for resuming partial data chunks
+        </description>
+    </param>
+    
+    <param name="fileSize" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>
+        Full Size of file. sends in first OnPutFile notification if file is splited into many PutFiles
```
Comment (jordynmackool) [inline proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md]: Update to:
Full Size of file. **Sends** in first OnPutFile notification **if the file is split** into many PutFiles

Or can it be:
Full Size of file is **sent** in first OnPutFile notification **if the file is split** into many PutFiles

Comment (NicoleYarroch) [reply proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md]: @jordynmackool I made the revisions. I created a new section for updating the documentation so the Core team knows they have to update the description for the `fileSize` parameter.

Comment (NicoleYarroch) [reply proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md]: @jordynmackool Do you want me to remove the above change and **not** make the update to documentation or do you want me to remove the above change but still make the update to the documentation?

Comment (jordynmackool) [reply proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md]: @NicoleYarroch make the noted change I commented above only. Then create an issue in the core repo about the documentation core update.

**Comment by NicoleYarroch** [comment]
@jordynmackool The proposal has been reverted to its original state.

**Pull request merged**